projects/04-burger-queen/README.pt-BR.md

```diff
@@ -0,0 +1,415 @@
+
+2. Faça um _fork_ deste repositório (no GitHub).
+
+3. Clone seu _fork_ no seu computador:
+
+   ```sh
+   git clone git@github.com:<tu-usuario-de-github>/<cohortid>-burger-queen.git
+   cd <cohortid>-burger-queen
+   ```
+
+4. Crie uma branch da `master` para começar a trabalhar. Por exemplo:
+
+   ```sh
+   git checkout -b develop
+   ```
+
+5. Crie um projeto no [Firebase](https://firebase.google.com/)
+
+6. Habilite o Firestore (_começar em modo bloqueado_) nas "Bases de Dados" de
+   [Firebase console](https://console.firebase.google.com/).
+
+7. Instale o utilitário de linha de comando do Firebase:
+
+   ```sh
+   npm i -g firebase-tools
+   ```
+
+8. Adicione o ambiente de produção para fazer o deploy:
+
+   ```sh
+   firebase use --add
+   ```
+
+9. Instale dependências de cloud functions:
+
+   ```sh
+   # usando yarn
+   cd functions && yarn && cd ..
+   # alternativamente, usando npm
+   cd functions && npm install && cd ..
+   ```
+
+10. Rode o comando: `firebase deploy`
+
+11. Neste ponto, você pode começar com o _front-end_ :wink:
+
+***
+
+Nota para a utilização do `create-react-app`:
+
+Se você tentar usar o `create-react-app` no diretório do projeto, você receberá
+um erro dizendo que há arquivos que podem apresentar um conflito. Para evitar
+esse problema você pode criar um novo aplicativo usando `create-react-app` e a
+partir daí _ junte com a pasta do projeto:
+
+```sh
+# se estava na pasta do projeto, fomos para a pasta acima
+cd ..
+
+create-react-app burger-queen-tmp
+cp -r burger-queen/* burger-queen-tmp/
+cp -r burger-queen-tmp/.gitignore burger-queen-tmp/* burger-queen/
+rm -rf burger-queen-tmp
+cd burger-queen
+```
+
+### Outros recursos
+
+#### Frameworks / bibliotecas
+
+* [React](https://reactjs.org/)
+* [Vue](https://vuejs.org/)
+* [Angular](https://angular.io/)
+
+#### Ferramentas
+
+* [npm-scripts](https://docs.npmjs.com/misc/scripts)
+* [Babel](https://babeljs.io/)
+* [webpack](https://webpack.js.org/)
+
+#### PWA
+
+* [Seu primeiro Progressive Web App - Google
+  developers](https://developers.google.com/web/fundamentals/codelabs/your-first-pwapp/?hl=es)
+* [Progressive Web Apps -
+  codigofacilito.com](https://codigofacilito.com/articulos/progressive-apps)
+* [offlinefirst.org](http://offlinefirst.org/)
+* [Usando Service Workers -
+  MDN](https://developer.mozilla.org/pt-BR/docs/Web/API/Service_Worker_API/Using_Service_Workers)
+* [Como habilitar dados sem conexão - Firebase
+  Docs](https://firebase.google.com/docs/firestore/manage-data/enable-offline?hl=es-419)
+
+#### Serverless
+
+* [Serverless Framework 1.0 - Pagar.me
+  Talks](https://www.youtube.com/watch?v=2oNovfw3V08)
+* [Firebase](https://firebase.google.com/)
+* [Serverless Architectures - Martin
+  Fowler](https://www.martinfowler.com/articles/serverless.html)
+
+#### Cloud functions
+
+* [Cloud functions - Firebase
+  Docs](https://firebase.google.com/docs/functions/?hl=es-419)
+
+<!-- ## Checklist
```

Comment:
Hey @paclaraujo, I think we can remove these comments as they were commented out from the Spanish version 11 months ago with not so much context: https://github.com/Laboratoria/bootcamp/commit/e20b6233869ed8e816f2dceab77af357bb294b25#diff-30a449096eab66b431fd16b28474f9e4R392

Reply:
Done!

Reply:
I meant from the BR version too!

Reply:
hahahahahaha sooo sorry. But know I hope it's ok !